day13/cruise-control.py

```diff
@@ -0,0 +1,20 @@
+def maxSpeed(horses, totalDist):
```

Comment:
i have nits throughout this function about spacing around operators: lines like

    time = (totalDist-start)*1.0/speed

should be

    time = (totalDist - start) * 1.0 / speed